test/compute_eigvec_from_eigval_lopcg.jl

```diff
@@ -9,16 +9,18 @@ import NonlinearEigenproblems.NEPCore.compute_Mlincomb
 
 @testset "compute eigvec lopcg" begin
     @testset "dep0_sparse" begin
-        nep=nep_gallery("dep0_sparse",100);nept=DEP([nep.A[1]',nep.A[2]'],nep.tauv);
+        nep = nep_gallery("dep0_sparse", 100);
+        nept = DEP([copy(nep.A[1]'), copy(nep.A[2]')], nep.tauv);
```

Comment:
Materializing the adjoints here cut the allocated memory in half.

Reply:
Great. Good find. The same problem probably exists in all methods that use a "transposed NEP" usually called nept. It should be in infbilanczos and rfi.

Reply:
Updated. I only found it in those two.